code-experiments/build/matlab/my_optimizer.m

```diff
@@ -6,5 +6,20 @@ function my_optimizer (problem, lower_bounds, upper_bounds, budget)
     cocoEvaluateFunction(problem, x);
     if cocoProblemGetNumberOfConstraints(problem) > 0
         cocoEvaluateConstraint(problem, x)
+        cocoProblemGetName(problem)
     end
 end
+
+%
+% asma: The code below calls MATLAB solver, fmincon, on bbob-constrained testbed
+% cons is defined in cons.m
+%
+
+% function my_optimizer (problem, lower_bounds, upper_bounds, budget)
```

Comment:
Now in a separate file (my_fmincon.m). I kept the comments for the moment as I find them helpful to give a quick overview of what is done by the code, but I can delete them if you prefer.